modules/cluster/src/main/java/io/fluo/cluster/command/MiniFluoCommand.java

```diff
@@ -77,26 +75,25 @@ public static void main(String[] args) {
       for (String logger : new String[] {Logger.ROOT_LOGGER_NAME, "io.fluo"}) {
         ((Logger) LoggerFactory.getLogger(logger)).setLevel(Level.ERROR);
       }
-    } else if (command.equalsIgnoreCase("classpath")) {
-      AppRunner.classpath("fluo", fluoHomeDir, remainArgs);
-      return;
     } else if (command.equalsIgnoreCase("wait")) {
       ((Logger) LoggerFactory.getLogger(FluoConfiguration.class)).setLevel(Level.ERROR);
     }
 
-    FluoPath cu = new FluoPath(fluoHomeDir, appName);
-
-    MiniAppRunner runner = new MiniAppRunner(cu.getAppConfiguration());
+    FluoInstall fluoInstall = new FluoInstall(fluoHomeDir);
+    MiniAppRunner runner = new MiniAppRunner();
 
     switch (command.toLowerCase()) {
+      case "classpath":
+        runner.classpath(fluoHomeDir, remainArgs);
+        break;
       case "scan":
-        runner.scan(chooseConfig(runner), remainArgs);
+        runner.scan(chooseConfig(fluoInstall.resolveFluoConfiguration(appName)), remainArgs);
```

Comment:
If we were to print out which config was used, in the case of mini it might not make sense to do it in resolveFluoConfiguration